spec/variables.md

```diff
@@ -9,19 +9,20 @@
 * [Semantics](#semantics)
   * [Executing a Variable Declaration](#executing-a-variable-declaration)
   * [Evaluating a Variable](#evaluating-a-variable)
-  
+
 ## Syntax
 
 <x><pre>
-**Variable**            ::= '$' Identifier | NamespacedVariable
+**Variable**            ::= VariableName | NamespacedVariable
+**VariableName**        ::= '$' Identifier
 **NamespacedVariable**  ::= Identifier '.$' [PublicIdentifier][]
 **VariableDeclaration** ::= Variable ':' Expression ('!global' | '!default')*
 </pre></x>
 
 [PublicIdentifier]: modules.md#syntax
 
 No whitespace is allowed after the `$` or before or after the `.$` in
-`Variable`. Each of `!global` and `!default` is allowed at most once in
+`VariableName`. Each of `!global` and `!default` is allowed at most once in
```

Comment:
`.$` doesn't appear in `VariableName`, so this sentence should probably be rewritten slightly.

Reply:
hmm, indeed. I missed the part about ``.$`` which should refer to `NamespacedVariable`